lib/_footer.erb

```diff
@@ -22,7 +22,7 @@
 
     <%- if !@site_map['Plugins'].nil? -%>
       <%- @site_map['Plugins'].each do |categories| -%>
-        <%- if categories.key?("Google Analytics") -%>
+        <%- if categories.key?("Google Analytics") && !categories["Google Analytics"].nil? -%>
```

Comment:
Are we missing GA if its set in the yml file before this nil check was added?

Reply:
No, we weren't missing GA before, but this is an additional safety check to prevent writing an erroneous script if the user provides the `Google Analytics` key on the yml file but doesn't provide a value for it.